module6/project/index.md

```diff
@@ -0,0 +1,108 @@
+
+The project will be split into 5 phases:
+* Ideation
+* Proof of Concept
+* Inception
+* Development
+* Presentation/Documentation
+
+### Project Requirements
+Your project must implement a new technology.  You may choose from the following list:
+* Create a Backend API using GraphQL
+* Use websockets rather than traditional HTTP/HTTPS
+* Deploy your application
+* Build a new feature using a new-to-you language or framework
+* Other (please discuss with an instructor)
+
+Throughout the project, you will have small group standups.  Your small group will be your project accountabilibuddies.  During each standup you should:
+* Review your current project board
+* Discuss what you accomplished the previous day
+* Outline what you want to accomplish this day
+* Discuss any blockers you are experiencing, and help each other form strategies for moving past the blockers
+  * These strategies could be additional research, posting in code-help, and/or reaching out to an instructor
+
+## Ideation
+
+During this phase, you should determine which new technology you will be using, and how it will improve your application.
+
+Complete the New Feature Template below - we will discuss your proposal during your **first project checkin**.  If you are planning to use a technology other than one of the list above, reach out to an instructor _before_ your first checkin.
+
+```
+## [Feature Name]
+### Description
+2 - 3 sentences that explains what the new feature will do, or how it will improve your application
+
+### New Technology
+* What new technology will you be using?
+* List 3 resources that will help you learn this new technology (include links)
+
+### Anticipated Challenges
+* What challenges do you forsee in implementing this feature?
+* When will you know that you are in unproductive struggle?
+* What strategies will you use to move forward?
+```
+
+## Proof of Concept
+
+The first 2 weeks of the project will be dedicated to 'Proof of Concept'.  During this time, you will learn as much as you can about your new technology, AND implement it outside of your project.  Only after successful implementation of your proof of concept will you be invited to move forward.
+
+You must demonstrate progress toward this goal at your **second project checkin**.  At that time, your instructor may advise you to choose a new technology, and start the ideation process over.
+
+#### An Example
+Let's say you choose to Deploy Your Application as your new technology.  During the 2 week proof-of-concept period, you will need to demonstrate that you can deploy a database-connected .NET web application.  If at your 2nd project checkin you have not demonstrated that you can deploy a single-endpoint web application (without a database), you may be asked to reconsider your new technology.
+
+#### TIL
+As part of your project documentation, you will be expected to include a TIL type write-up on your new technology.  This can be added to your TIL repo on GitHub.
+
```

Comment:
I like the TIL structure! We could also put something here to encourage folks to share this on LinkedIn.

I like the idea of also doing a presentation about this technology for peers. Maybe small groups where we break up folks who researched the same technology.